src/test/java/io/confluent/connect/elasticsearch/integration/README.md

```diff
@@ -0,0 +1,12 @@
+The Integration tests are built to run with `maven-surefire-plugin` in conjuction with 
+`docker-maven-plugin` by `io.fabric8` to set up docker containers for Elasticsearch 
+(embedded nodes are no longer supported or recommended by Elasticsearch).
+
+To run the tests in their entirety, use `mvn verify`.
```

Comment:
wouldn't `mvn integration-test` work too?

Reply:
According to the failsafe docs, "NOTE: when running integration tests, you should invoke Maven with the (shorter to type too) `mvn verify` rather than trying to invoke the integration-test phase directly, as otherwise the post-integration-test phase will not be executed." I can add this language to the README